### PR TITLE
Add Messaging Bridge Pattern Sample

### LINKED ISSUE #21: Create a sample for Messaging Bridge pattern
Create a sample for [Messaging Bridge](https://www.enterpriseintegrationpatterns.com/patterns/messaging/MessagingBridge.html)

### PR DESCRIPTION
**Pull request by DimuthuMadushan**
## Purpose
Fixes: [#21](https://github.com/ballerina-guides/enterprise-integration-patterns/issues/21)

### CODE REVIEW

File: messaging-bridge/messaging-bridge.bal [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import ballerina/graphql;
+import ballerina/http;
+
+type Project record {
```
Comment (LakshanWeerasinghe) [inline messaging-bridge/messaging-bridge.bal]: This needs to be a closed record.

File: messaging-bridge/tests/case-01.http [hunk 2]
```diff
@@ -0,0 +1,7 @@
+POST http://localhost:8080/api/v1
+content-type: application/json
```
Comment (LakshanWeerasinghe) [inline messaging-bridge/tests/case-01.http:2]:
```suggestion
```
By default it is application/json

Comment (DimuthuMadushan) [reply messaging-bridge/tests/case-01.http:2]: No right? It's `text/plain`.

File: messaging-bridge/messaging-bridge.bal [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    string totalHours;
+};
+
+final http:Client zoho = check new("http://zohoapis.com.balmock.io");
+
+service /api/v1 on new graphql:Listener(8080) {
+
+    resource function get project(string projectID, string organizationID) returns Project|error {
+        return zoho->/books/v3/projects/[projectID].post({organization_id: organizationID});
+    }
```
Comment (manuranga) [inline messaging-bridge/messaging-bridge.bal:31]: let's add another resource.

File: messaging-bridge/messaging-bridge.bal [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import ballerina/graphql;
+import ballerina/http;
+
+type Project record {|
+    string projectID;
+    string projectName;
+    string description;
+    string customerName;
+    Task[] tasks;
+|};
```
Comment (manuranga) [inline messaging-bridge/messaging-bridge.bal]:
```
type ProjectRequest record {|
    string projectName;
    string description;
    string customerName;
|}

type Project record {|
    *ProjectRequest;
    string projectID;
    Task[] tasks;
|};
```

File: messaging-bridge/messaging-bridge.bal [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+final http:Client zoho = check new("http://zohoapis.com.balmock.io");
+
+service /api/v1 on new graphql:Listener(8080) {
+
+    resource function get project(string organizationID, string projectID) returns Project|error {
+        return zoho->/books/v3/projects/[projectID].get(organization_id = organizationID);
+    }
+
+    remote function createProject(ProjectInput projectInput, string organizationID) returns Project|error {
```
Comment (manuranga) [inline messaging-bridge/messaging-bridge.bal]: `remote function createProject(string organizationID, ProjectRequest projectRequest) returns Project|error {`

**Review by manuranga — APPROVED** [review]

**Pull request merged**